src/components/NewNotesButton/index.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useState, useEffect } from 'react'
+import UserAvatar from '@/components/UserAvatar'
+import storage from '@/services/local-storage.service'
+import { TNewNotesButtonPosition } from '@/types'
+
+const styles = {
+  settingsToggle: 'flex items-center justify-between gap-4 p-4',
+  label: 'text-sm font-medium',
+  switchBase: 'relative inline-flex h-6 w-11 items-center rounded-full transition-colors',
+  switchActive: 'bg-primary',
+  switchInactive: 'bg-gray-300',
+  switchThumb: 'inline-block h-4 w-4 transform rounded-full bg-white transition-transform',
+  thumbActive: 'translate-x-6',
+  thumbInactive: 'translate-x-1',
+  fixedContainerTop: 'sticky top-[3.5rem] w-full flex justify-center my-2 z-30',
+  fixedContainerBottom: 'fixed bottom-4 left-0 right-0 w-full flex justify-center z-[9999]',
+  button:
+    'flex items-center bg-purple-600 hover:bg-purple-700 text-white px-4 py-2 rounded-full text-sm font-medium shadow-lg',
+  avatarsContainer: 'flex items-center mr-1',
+  avatarWrapper: 'relative -mr-2.5 last:mr-0',
+  avatar:
+    'w-7 h-7 rounded-full border-2 border-purple-600 overflow-hidden flex items-center justify-center bg-background',
+  countText: 'whitespace-nowrap ml-1'
+}

```

Comment:
Write Tailwind CSS directly in the className to maintain a consistent code style.